runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -441,11 +516,137 @@ public void onWidgetLoaded(WidgetLoaded event)
 	public void onScriptPostFired(ScriptPostFired event)
 	{
 		// GE offers setup init
-		if (event.getScriptId() != ScriptID.GE_OFFERS_SETUP_BUILD)
+		if (event.getScriptId() == ScriptID.GE_OFFERS_SETUP_BUILD)
+		{
+			rebuildGeText();
+		}
+		else if (event.getScriptId() == ScriptID.GE_ITEM_SEARCH && config.highlightSearchMatch())
+		{
+			highlightSearchMatches();
```

Comment:
What is the point of this without fuzzy matches? This seems like it would not be useful since the match would always contain the literal search